README.md

```diff
@@ -235,6 +235,7 @@ To the extent possible under law, [Vitali Fokin](https://github.com/quozd) has w
 * [clipr](https://github.com/nemec/clipr) - A CLI library inspired by Python's argparse that transforms a command line into a strongly-typed object. It supports custom argument types, automated (and localized) help generation, and a variety of ways to store parsed arguments.
 * [CliWrap](https://github.com/Tyrrrz/CliWrap) - Wrapper for command line interfaces.
 * [Colorful.Console](https://github.com/tomakita/Colorful.Console) - Colorful console output.
+* [CommandDotNet](https://github.com/bilal-fazlani/commanddotnet) - Model your console app using C# in a composable manner.  Define commands with methods.  Define subcommands with properties or nested classes.  Extensible parsing and command execution.   
```

Comment:
Please remove double spaces

Reply:
Done.  That's an old typing habit that just hasn't died yet. :)

Reply:
How was that ever a useful habbit?